src/ptf/testutils.py

```diff
@@ -2291,7 +2294,7 @@ def verify_no_other_packets(test, device_number=0, timeout=None):
         test.fail("A packet was received on device %d, port %r, but we expected no "
                   "packets.\n%s" % (result.device, result.port, result.format()))
 
-def verify_packets(test, pkt, ports=[], device_number=0):
+def verify_packets(test, pkt, ports=[], device_number=0, timeout=None):
```

Comment:
maybe just add a comment explaining that timeout will be passed as is for each individual verify call.